test/blackbox-tests/test-cases/pkg/depopts/depopts-with-conflicting-constraints.t

```diff
@@ -0,0 +1,27 @@
+We test depopts with conflicting constraints to see which one the solver will
+prefer if any:
+
+  $ . ../helpers.sh
+  $ mkpkg foo 1
+  $ mkpkg foo 2
+
+  $ mkpkg bar <<'EOF'
+  > depopts: [ "foo" {= "1"} ]
+  > EOF
+
+  $ mkpkg baz <<'EOF'
+  > depopts: [ "foo" {= "2"} ]
+  > EOF
+
+We don't currently support depopts so they are both omitted by omission.
+
+  $ solve bar baz
+  Solution for dune.lock:
+  - bar.0.0.1
+  - baz.0.0.1
+
+In the future, we could support "local depopts" that are only in scope for
```

Comment:
This description is wrong. It will never be possible to install multiple depopts side by side and all 4 packages. What is possible is selecting a version of `foo` to satisfy `bar` (or `baz`), and then building `baz` (or `bar`) without the depopt.

Reply:
OK that's 3, in my suggestions above. I think I'll update the description to 2, as that is what you are describing.

Reply:
3. is not possible. You will get libraries that cannot be linked together

Reply:
OK, I've updated the description to mention 2 instead.